extensions/resteasy-classic/resteasy/runtime/src/main/java/io/quarkus/resteasy/runtime/AuthenticationCompletionExceptionMapper.java

```diff
@@ -6,16 +6,25 @@
 
 import org.jboss.logging.Logger;
 
+import io.quarkus.runtime.LaunchMode;
 import io.quarkus.security.AuthenticationCompletionException;
 
 @Provider
 public class AuthenticationCompletionExceptionMapper implements ExceptionMapper<AuthenticationCompletionException> {
 
     private static final Logger log = Logger.getLogger(AuthenticationCompletionExceptionMapper.class.getName());
+    private final boolean addResponseBody;
```

Comment:
IMHO it is not necessary to have this boolean flag since in devmode we indeed would like to return exception messages

Reply:
Sorry, `LaunchMode.isDev` you introduced here is very compact way of checking, but if you prefer to keep this boolean, please rename to `includeExceptionMessage`, as I thought initially you were planning to include some error response data from OIDC providers, etc

Reply:
it's less verbose, I tried to make it fast but I think it probably had no effect at all